src/collections/views/covid-19-vaccination-centre-data-monitor-accessibility-map.md

```diff
@@ -19,11 +19,25 @@ This map demonstration was created to show how these data gaps can be addres
 1. A base map with markers over the testing centres 
 2. Each marker will be clickable and include the following: 
 
-* Accessibility features  
-* Commuting options to the location 
-* Contact information 
-* Street view/photos linking to Google Maps 
-* Comments and insights   
+* Accessibility features
+* Commuting options to the location
+* Contact information
+* Street view/photos linking to Google Maps
+* Comments and insights
+
+## Map demonstration
```

Comment:
The section above uses the same title. The section above probably could use another title such as "Map features".

@uttaraghodke, @amb26, what do you think?